share/functions/cd.fish

```diff
@@ -3,10 +3,17 @@
 #
 function cd --description "Change directory"
     set -l MAX_DIR_HIST 25
+    set -l options h/help L P
```

Comment:
this should include the long options: `P/physical`, and `L/logical`

However, do we need the "logical" option at all? I feel like we can remove it because it's the default, or do people expect that option?

Reply:
I think it's good to have it. The other shell have this option.